doc/Caliptra.md

```diff
@@ -272,10 +276,29 @@ Caliptra is among the first uncore microcontrollers taken out of reset by the po
 
 See [Error Reporting and Handling](#error-reporting-and-handling) for details about Caliptra and SoC firmware load and verification error handling.
 
-*Figure 3: Caliptra boot flow*
+*Figure 3: Passive Caliptra boot flow*
 
 ![](./images/Boot_flow.png)
 
+**Subsystem Mode Boot Flow**
+
+MCU (Manufacterer Control Unit), that is holds platform & SOC specific FW and Caliptra are among the first microcontrollers taken out of reset by the power-on reset logic. Caliptra is responsible for the start of the firmware chain-of-trust with the immutable component of the MCU ROM. After the Caliptra ROM completes initialization, it provides a "stash measurement" API and callback signals for MCU ROM (subsystem mode) to proceed with the boot process. Caliptra ROM supports stashing of at most eight measurements prior to the boot of Caliptra RT firmware.  Then Caliptra FW is loaded through OCP streaming boot flow. Any security-sensitive code (eg. PLL programming) or configuration (eg. Fuse based Patching) loaded by the MCU prior to Caliptra firmware boot must be stashed within Caliptra. If the MCU exceeds Caliptra ROM's measurement stash capacity, attestation must be disabled until the next cold reset. 
+
+Note: This is extremely high level flow, please see the Subsystem Mode Section below for next level specifics.
+
+The high level boot process is as follows:
+
+1. Hardware executes SoC power-on reset logic. This logic starts the execution of MCU ROM and Caliptra ROM.
+2. Recovery interface is gated until ready for recovery is written into recovery interface registers from Caliptra ROM. This happens at the same time as passive mode's ready_for_fw signal.
```

Comment:
nit: Should this be formatted as `ready_for_recovery` to refer to a specific signal?

Reply:
Reason I didnt combine everything is because it is not a wire but an explicit register write